infra/gcp/ensure-release-projects.sh

```diff
@@ -116,18 +118,26 @@ for PROJECT; do
     # Enable GCB and Prow to build and push images.
 
     # Let project writers use GCB.
-    for group in ${ADMINS} ${WRITERS}; do
+    for group in ${RELEASE_ADMINS} ${RELEASE_MANAGERS}; do
         color 6 "Empowering ${group} as GCB editors"
         empower_group_for_gcb "${PROJECT}" "${group}"
     done
 
     # Let prow trigger builds and access the scratch bucket
-    color 6 "Empowering Prow"
-    empower_prow "${PROJECT}" "${GCB_BUCKET}"
+    serviceaccount="${GCB_BUILDER_SVCACCT}"
+    principal="serviceAccount:${serviceaccount}"
+
+    color 6 "Ensuring ${serviceaccount} can use GCB in project: ${PROJECT}"
+    ensure_project_role_binding "${PROJECT}" "${principal}" "roles/cloudbuild.builds.builder"
+    ensure_gcs_role_binding "${GCB_BUCKET}" "${principal}" "objectCreator"
+    ensure_gcs_role_binding "${GCB_BUCKET}" "${principal}" "objectViewer"
+
+    color 6 "Ensuring k8s-prow / test-infra-trusted can no longer use GCB in project: ${PROJECT}"
+    ensure_removed_google_prow_bindings "${PROJECT}" "${GCB_BUCKET}"
```

Comment:
I'm not sure but this part is applied once. Why do it ever time the script is used ?

Reply:
The idea is to remove this in a follow-up PR.  I was starting to get lazy by this point so I didn't go as far as making a "bindings to remove" array like I did in ensure-organizations.